internal/tracer/jaeger.go

```diff
@@ -0,0 +1,46 @@
+package tracer
+
+import (
+	"log"
+	"reflect"
+
+	"github.com/opentracing/opentracing-go"
+	"github.com/uber/jaeger-client-go"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+	jaegermetrics "github.com/uber/jaeger-lib/metrics"
+)
+
+func configureJaeger(svcName string, version string) (opentracing.Tracer, error) {
+	cfg, err := jaegercfg.FromEnv()
+	cfg.ServiceName = svcName
+	if err != nil {
+		return nil, err
+	}
+	cfg.Tags = append(cfg.Tags, opentracing.Tag{Key: "service.version", Value: version})
+	if reflect.DeepEqual(cfg.Sampler, &jaegercfg.SamplerConfig{}) {
+		// Default sampler configuration for when it is not specified via
+		// JAEGER_SAMPLER_* env vars. In most cases, this is sufficient
+		// enough to connect to Jaeger without any env vars.
+		cfg.Sampler.Type = jaeger.SamplerTypeConst
+		cfg.Sampler.Param = 1
```

Comment:
Probably worth adding a small comment on why it's `1` here.

Reply:
I'm not sure either, this is just moved code (the jaeger setup here is unchanged) - I'll take a look and try to comment on it

Reply:
https://github.com/sourcegraph/zoekt/pull/413/commits/f87e6a2611c2164c3702b8e60fa18ef182852855